test/parallel/test-buffer.js

```diff
@@ -1189,3 +1189,7 @@ assert.throws(function() {
 assert.throws(function() {
   new Buffer(null);
 }, /must start with number, buffer, array or string/);
+
+// Test the zero fill option
+const zeroFilledBuffer = Buffer(5, true);
+assert.equal(0, Buffer.compare(zeroFilledBuffer, Buffer([0, 0, 0, 0, 0])));
```

Comment:
I'm saying that if the actual zero fill failed while creating `zeroFilledBuffer` it may still be zero filled. The allocated memory for a new Buffer may not have anything there. So the fill could fail but this test may succeed.